### PR TITLE
Convert ansible repo to a template

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This will help us to easily upgrade to higher ansible versions.

### CODE REVIEW

**Comment by chaitanyaenr** [comment]
@wabouhamad @mffiedler PTAL,

File: image_provisioner/playbooks/roles/repo-install/templates/etc/yum.repos.d/rhel7-server-ansible.repo.j2 [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [rhel-7-server-ansible]
 name=rhel-7-server-ansible-2.4-rpms
```
Comment (mffiedler) [inline image_provisioner/playbooks/roles/repo-install/templates/etc/yum.repos.d/rhel7-server-ansible.repo.j2]: Super nitpicky - can we change the "name=" to something generic ?

**Comment by chaitanyaenr** [comment]
Will quickly test this change and remove the do-not-merge label,

**Comment by chaitanyaenr** [comment]
@wabouhamad @mffiedler this PR is ready for review.

**Pull request merged**